libs/labelbox/src/labelbox/data/annotation_types/collection.py

```diff
@@ -40,47 +40,6 @@ def _assign_ids(label: Label):
         self._fns["assign_feature_schema_ids"] = _assign_ids
         return self
 
-    def add_url_to_data(
-        self, signer: Callable[[bytes], str]
-    ) -> "LabelGenerator":
-        """
-        Creates signed urls for the data
-        Only uploads url if one doesn't already exist.
-
-        Args:
-            signer: A function that accepts bytes and returns a signed url.
-        Returns:
-            LabelGenerator that signs urls as data is accessed
-        """
-
-        def _add_url_to_data(label: Label):
-            label.add_url_to_data(signer)
-            return label
-
-        self._fns["add_url_to_data"] = _add_url_to_data
-        return self
-
-    def add_to_dataset(
```

Comment:
Same concept of add_url_to_data. This was only used in TextData and GenericDataRow does not support this method

Reply:
I can add it to GenericDataRow but I think methods like this were never used and are complicated